charts/airbyte-server/templates/deployment.yaml

```diff
@@ -236,9 +255,11 @@ spec:
       {{ toYaml .Values.global.extraContainers | indent 8 }}
       {{- end }}
       volumes:
+      {{- if eq .Values.deploymentMode "oss" }}
       - name: gcs-log-creds-volume
         secret:
           secretName: {{ ternary (printf "%s-gcs-log-creds" ( .Release.Name )) (.Values.global.credVolumeOverride) (eq .Values.global.deploymentMode "oss") }}
+      {{- end }}
       {{- if .Values.extraVolumes }}
```

Comment:
if we are not `deploymentMode: oss` and we also don't have `extraVolumes` we will have a `volumes` key with no value.  Is that fine?

Reply:
Yep that's totally fine, it'll act as an empty list or dict when yaml will be transformed into JSON before sending request to k8s api